common/data_refinery_common/utils.py

```diff
@@ -235,14 +244,41 @@ def get_readable_affymetrix_names(mapping_csv: str="config/readable_affymetrix_n
         for line in reader:
             # Skip the header row
             # Lines are 1 indexed, #BecauseCSV
-            if reader.line_num is 1:
+            if reader.line_num == 1:
                 continue
 
             READABLE_PLATFORM_NAMES[line[1]] = line[0]
 
     return READABLE_PLATFORM_NAMES
 
 
+def get_affymetrix_annotation_package_name_overrides(
+    overrides_csv: str = "config/affymetrix_annotation_package_name_overrides.csv"
+) -> Dict:
+    """
+    Loads the mapping from annotation package name to internal accession for Affymetrix platforms.
+    CSV must be in the format:
+    Annotation Package Name | Inernal Accession
+    """
+    global ANNOTATION_PACKAGE_OVERRIDES
+
+    if ANNOTATION_PACKAGE_OVERRIDES is not None:
+        return ANNOTATION_PACKAGE_OVERRIDES
+
+    ANNOTATION_PACKAGE_OVERRIDES = {}
+    with open(overrides_csv, encoding='utf-8') as overrides_file:
+        reader = csv.reader(overrides_file, )
+        for line in reader:
```

Comment:
Is skipping preferred under our style guide to using `next(reader)` before starting the loop? I'd probably do the `next...` approach.

Reply:
Yeah I think that would make more sense too